src/data/online.json

```diff
@@ -491,5 +491,22 @@
 		"author": "albinowax",
 		"notes": "Web application hacking game via missions, based on real vulnerabilities.",
 		"badge": null
+	},
+	{
+		"url": "https://cyberscavengerhunt.com",
+		"name": "Cyber Scavenger Hunt",
+		"technology": [
+			"Javacript",
+			"React"
+		],
+		"references": [
+			{
+				"name": "download",
+				"url": "https://github.com/arthurakay/cyberscavengerhunt"
+			}
+		],

```

Comment:
Is there a live one?

If not and this is meant to be locally hosted you should actually add it to the offline list.

Reply:
It's live at https://cyberscavengerhunt.com -- see line 496.

I wasn't entirely sure if I needed to add the GitHub code or not (you certainly can run it locally)

Reply:
Doh totally missed that, thanks!